wiki/FitNesseRoot/HsacExamples/SlimTests/BrowserTests/ScriptTest.wiki

```diff
@@ -1,42 +1,38 @@
-|show                                   |click if available|css=.registration-prompt-close                     |
-|show                                   |take screenshot   |overview                                           |
-|click                                  |Discography                                                           |
-|check                                  |page title        |!-Van Morrison | Album Discography | AllMusic-!    |
-|note                                   |close popup, if shown                                                 |
-|show                                   |click if available|css=.registration-prompt-close                     |
-|show                                   |take screenshot   |discography                                        |
-|check                                  |value of          |Year       |in row where|Album|is|Astral Weeks|1968|
-|note                                   |close popup, if shown                                                 |
-|show                                   |click if available|css=.registration-prompt-close                     |
-|note                                   |scroll top of discography on screen                                   |
-|scroll to                              |Filter Discography By                                                 |
-|click                                  |Astral Weeks                                                          |
-|show                                   |take screenshot   |album                                              |
-|show                                   |take screenshot   |cover_image|of          |click to embiggen         |
-|click                                  |click to embiggen                                                     |
-|                                       |wait for cover to appear                                              |
-|wait                                   |1                 |seconds                                            |
-|show                                   |take screenshot   |cover                                              |
+|script          |browser test                                                            |
+|open            |https://v4.practicesoftwaretesting.com                                  |
+|enter           |Pliers                              |as|Search                          |
+|press           |enter                                                                   |
+|wait for visible|css=[data-test='search_completed']                                      |
+|click           |Combination Pliers                                                      |
+|click           |Add to cart                                                             |
+|click           |css=[data-test='nav-cart']                                              |
```

Comment:
This, of course, works. But maybe for the 'accessibility' using screen readers of your site you should add an aria-label to the cart icon.
The element could then be found using this label...

Reply:
yep, makes sense...